### ISSUE TITLE
Loss is very high!

### ISSUE THREAD

**Issue by gauravsh0812** [PKU-YuanGroup/Chat-UniVi#67]
Hi there!

I have trained the model on my custom video-QA dataset. During the entire training, the loss seems very high (~20-30). I am not sure why it is so. Even after fine-tuning, it gets down to ~6. And, as expected the inference output was garbage and had no proper sentence at all. Can anyone share their experience if they have faced similar issues?
